test/system/155-partial-pull.bats

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# tests for podman partial pulls support
+#
+
+load helpers
+load helpers.network
+load helpers.registry
+
+###############################################################################
+# BEGIN filtering - none of these tests will work with podman-remote
+
+function setup() {
+    skip_if_remote "none of these tests work with podman-remote"
+
+    basic_setup
+    start_registry
+}
+
+# END   filtering - none of these tests will work with podman-remote
+###############################################################################
+# BEGIN actual tests
+# BEGIN primary podman push/pull tests
+
+@test "push and pull zstd chunked image" {
+    image1=localhost:${PODMAN_LOGIN_REGISTRY_PORT}/img1-$(safename)
+
+    globalargs="--pull-option enable_partial_pulls=true"
```

Comment:
@giuseppe I can’t see any consumer of this option (set twice in this PR). I know about `enable_partial_images` .

Is this a typo, or am I missing something?

Reply:
Filed #24686 to fix that.